### ISSUE TITLE
aruco build fails in opencv_contrib-4.x

### ISSUE THREAD

**Issue by johncblacker** [opencv/opencv_contrib#3536]
##### Issue submission checklist

 - [ ] I report the issue, it's not a question
   
 - [ ] I checked the problem with documentation, FAQ, open issues,
       forum.opencv.org, Stack Overflow, etc and have not found any solution
   
 - [ ] I updated to the latest OpenCV version and the issue is still there
   
 - [ ] There is reproducer code and related data files: videos, images, onnx, etc

**Comment by johncblacker** [comment]
Figured out what I had to do.  
1.) downloaded the opcncv_contrib-4.x repository  # that gave me everything
2.) issued: git tags -l   # found the tag for the particular release that works - in my case 4.5.5
3.) issued: git checkout tags/4.5.5

Then I looked at the aruco.hpp file - specifically at the "include" statements and they were correct.

Another method I found was to list the tags on github, select the 4.5.5 version and download the zip version of the code; but this is NOT a git repository...
So, problem solved.
